general-rules.adoc

```diff
@@ -129,14 +164,6 @@ https://github.com/robocup-junior
 licensing rules. The focus should remain on personal growth and
 learning.
 
-==== Plagiarism Guidelines
```

Comment:
@dbscoach any particular reason why this is being removed? It was one of the more prominent additions this year.